oauth-authorization-server/pom.xml

```diff
@@ -56,13 +57,14 @@
 		<dependency>
 			<groupId>org.springframework.security.oauth</groupId>
 			<artifactId>spring-security-oauth2</artifactId>
-			<version>${oauth.version}</version>
+			<version>2.3.3.RELEASE</version>
```

Comment:
I'm not quite sure about this, but you might be able to skip specifying the version. It will use the starter-parent version.
This assures all included spring dependencies are version compatible.
it's just a note, and I'm not really sure if I'm correct, you can leave it as it is.

Reply:
Hm, my InteliiJ could not figure out the versions on its own, so I decided to add specific ones. Will leave it as it is for now and we can test it again and remove them if needed.